src/backward.h

```diff
@@ -4029,7 +4029,7 @@ class SignalHandling {
 #    elif defined(__arm__)
         error_addr = reinterpret_cast<void*>(uctx->uc_mcontext.arm_pc);
 #    elif defined(__aarch64__)
-        error_addr = reinterpret_cast<void*>(uctx->uc_mcontext.pc);
+        error_addr = reinterpret_cast<void*>(uctx->uc_mcontext->__ss.__pc);
```

Comment:
Confirmed that this works on my M1 Mac. Before merging, I also want to check that this works on Linux aarch64, as that's not currently covered by the test workflows